.npmignore

```diff
@@ -1,8 +1,7 @@
-build/
 docs/
-src/
 test/
 coverage/
 circle.yml
 tslint.json
+tsconfig.json
```

Comment:
nit: I prefer how blueprint-chart has a `yarn dist` command that copies the relevant stuff into a `dist/` folder; that way we whitelist the things that get packaged